osu.Game/Screens/Footer/ScreenFooter.cs

```diff
@@ -269,22 +340,34 @@ private void updateColourScheme(int hue)
 
             background.FadeColour(colourProvider.Background5, 150, Easing.OutQuint);
 
-            foreach (var button in buttonsFlow)
+            foreach (var button in leftButtonsFlow)
                 button.UpdateDisplay();
         }
 
-        private void makeButtonAppearFromLeft(ScreenFooterButton button, int index, int count, float startDelay)
+        private void makeLeftButtonAppearFromLeft(ScreenFooterButton button, int index, int count, float startDelay)
             => button.AppearFromLeft(startDelay + (count - index) * delay_per_button);
 
-        private void makeButtonAppearFromBottom(ScreenFooterButton button, int index)
+        private void makeLeftButtonAppearFromBottom(ScreenFooterButton button, int index)
             => button.AppearFromBottom(index * delay_per_button);
 
-        private void makeButtonDisappearToRight(ScreenFooterButton button, int index, int count, bool expire)
+        private void makeLeftButtonDisappearToRight(ScreenFooterButton button, int index, int count, bool expire)
             => button.DisappearToRight((count - index) * delay_per_button, expire);
 
-        private void makeButtonDisappearToBottom(ScreenFooterButton button, int index, int count, bool expire)
+        private void makeLeftButtonDisappearToBottom(ScreenFooterButton button, int index, int count, bool expire)
             => button.DisappearToBottom((count - index) * delay_per_button, expire);
 
+        private void makeRightButtonAppearFromLeft(ShearedButton button, float startDelay)
+            => button.AppearFromLeft(startDelay);
+
+        private void makeRightButtonAppearFromBottom(ShearedButton button)
+            => button.AppearFromBottom();
+
+        private void makeRightButtonDisappearToRight(ShearedButton button, bool expire)
+            => button.DisappearToRight(expire);
+
+        private void makeRightButtonDisappearToBottom(ShearedButton button, bool expire)
+            => button.DisappearToBottom(expire);
```

Comment:
oh and this is also pretty terrible but i am hoping that if the buttons can stop animating themselves for whatever reason we can just get rid of this entire block